project.clj

```diff
@@ -76,7 +76,7 @@
 
   :plugins [[lein-release "1.0.5" :exclusions [org.clojure/clojure]]]
 
-  :uberjar-name "puppet-server-release.jar"
+  :uberjar-name "puppetserver-release.jar"
```

Comment:
I think I was mostly thinking about upgrades - if we just change this, will we leave the old jar behind? I don't think I understand packaging enough on the various platforms to know that for sure.